crates/ra_hir/src/nameres/raw.rs

```diff
@@ -183,7 +184,18 @@ pub(super) struct Macro(RawId);
 impl_arena_id!(Macro);
 
 #[derive(Debug, PartialEq, Eq)]
-pub(super) struct MacroData {
+pub(super) enum MacroData {
+    MacroCall(MacroCallData),
+    DeriveAttr(DeriveData),
```

Comment:
I *think* that maybe we shouldn't single out the derive attribute at this stage and just store all attrs as token trees?